test/mz-debug/license_key

```diff
@@ -0,0 +1 @@
+eyJ0eXAiOiJKV1QiLCJhbGciOiJQUzI1NiJ9.eyJzdWIiOiIwMDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDAiLCJleHAiOjIwNTg5NzQ2ODgsIm5iZiI6MTc0MzYxNDY4OCwiaXNzIjoiTWF0ZXJpYWxpemUsIEluYy4iLCJhdWQiOiIwMDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDAiLCJpYXQiOjE3NDM2MTQ2ODgsImp0aSI6ImVkZGFmMDA0LWRjMWUtNDhjZi05Y2MxLTQxZDE1NDNkOTQwYSIsInZlcnNpb24iOjEsIm1heF9jcmVkaXRfY29uc3VtcHRpb25fcmF0ZSI6MTAwMC4wLCJhbGxvd19jcmVkaXRfY29uc3VtcHRpb25fb3ZlcnJpZGUiOnRydWUsImV4cGlyYXRpb25fYmVoYXZpb3IiOiJXYXJuIn0.a-O0xEfwgKKWkKCUoKF_odY_-ZbMN8t-fkLLfVvYkprD-uI2mFtWwVup9rRNDPdXO6M4aDkmTuTaX5Wt9jv1pPHDicICqPITtwCisvVRzV4IRcP09DinN-vooTcQy-ehG4smlpZ3CpWdig3CiJI4Sjv_QMTr6G8ETslZHCAtUfNpglSDXDCIBBujmiAQB6LpgTzNV6Uuq6t-FPKGAfhs3NkxSV_KGm-4LcoqhrqiLv6IotanwTTf7ZUeyW0HApdTbfFw6oBh_RORFULHcLojI8n5kAy2VIqzgc_ft2EpFNsw_L0XqhvECIfr-6f1oXErD7YdXNuEC2jdfWnapg3k7i61IbHyU6bo35tNw3AFynsx29UeeClmHdYW_UsyZAQmwjvW3pFo9Itaz1v9cVggZew_pa-tt6qXVvr9T6MnCbjj-zxJhAWyR4vRms817t4VgqHW9paa9l9aw4adWAkpzK3W0D1U0_lsCM-_918znZkmn0c7WxAFHAvjxob9d1xcJna06grsmzvCcFdtdlveqZB-s4nxDAB5rM5rm6vrLBf81x8UCuwA76NJxAmSXvKpVqS7iet_Crow90UPCOUoxWl2ql6HUXqVj4dl7nS43v42rFZGonjE4pL-A-7QYXfoqly0LoR-kCeBwHpKGJtDzBYBRl14-YGjH-y9HfOvBkI
```

Comment:
Ouch, I think we have to invalidate and recreate this key now for everyone. CC @doy-materialize 
I'll add license_key to .gitignore so this doesn't happen so easily again.

Reply:
which key is this? (where did it come from?)

Reply:
It's the materialize dev license key in 1password.

Reply:
https://github.com/MaterializeInc/materialize/pull/32144